convert_state.py

```diff
@@ -1,60 +1,65 @@
+"""
+Command line program to convert state names to abbreviations or convert abbreviations to names
+"""
 
+import json
 
 
-import json  # import json for json processing
 def main():
-    with open('data.json','r') as file:
-        data = file.read()
-        data = json.loads(data)
+    """
+    Read in JSON data and loop over
+    """
+    with open('data.json', 'r') as file:
+        state_names = file.read()
+        state_names = json.loads(state_names)
```

Comment:
Then state_names is overwritten with a dictionary.  Bad style to change the type of a variable. Languages like Java will prevent this, Python assumes you know what you are doing. This code works but it's bad style and confusing as to what the actual type of state_names is

Reply:
Made the change and pushed. Thanks! Didn't catch this in class.